lib/pack/localpack/packageserver.go

```diff
@@ -434,6 +436,32 @@ func (p *PackageServer) ConfigurePackage(loc loc.Locator, confLoc loc.Locator, a
 	return pack.ConfigurePackage(p, loc, confLoc, args, nil)
 }
 
+func (p *PackageServer) tryDeleteBlob(toDelete storage.Package) error {
```

Comment:
Simplest possible implementation based on the fact that deletes are rare - check for collisions on delete.